pydash/utilities.py

```diff
@@ -393,6 +394,34 @@ def noop(*args, **kargs):  # pylint: disable=unused-argument
     pass
 
 
+def nth_arg(pos):
+    """Creates a function that gets the argument at index n. If n is negative,
+    the nth argument from the end is returned.
+
+    Args:
+        pos (int): The index of the argument to return.
+
+    Returns:
+        function: Returns the new pass-thru function.
+
+    Example:
+
+        >>> func = nth_arg(1)
+        >>> func(11, 22, 33, 44)
+        22
+        >>> func = nth_arg(-1)
+        >>> func(11, 22, 33, 44)
+        44
+
+    .. versionadded:: TODO
+    """
+    def _nth_arg(*args):
+        """Returns argument at the given position :attr:`pos`."""
```

Comment:
If you're going to give the created function a docstring, then may as well replace `pos` in the description with the actual value given. Otherwise, you could skip the docstring. None of the other utility functions set a docstring on returned functions.